packages/playwright-core/src/client/elementHandle.ts

```diff
@@ -322,7 +283,7 @@ export async function convertInputFiles(files: string | FilePayload | string[] |
       error += ', please write it to a file and pass its path instead.';
     throw new Error(error);
   }
-  return { files: payloads };
+  return { payloads };
```

Comment:
Shouldn't you remove the check on the line 282 as well? Perhaps include the removal of this method and protocol field in the same PR?

Reply:
Done.